src/app/containers/Timestamp/timestampUtilities.js

```diff
@@ -12,31 +13,21 @@ export const isValidDateTime = dateTime => {
 
 export const leadingZero = val => (val < 10 ? `0${val}` : `${val}`);
 
-// 2019-03-22
-export const longNumeric = {
-  day: date => leadingZero(date.getDate()),
-  month: date => leadingZero(date.getMonth() + 1),
-  year: date => date.getFullYear(),
-  format: (d, m, y) => [y, m, d].join('-'),
-};
-
-// 22 March 2019
-export const shortAlphaNumeric = {
-  day: date => date.getDate(),
-  month: date => months[date.getMonth()],
-  year: date => date.getFullYear(),
-  format: (d, m, y) => [d, m, y].join(' '),
-};
-
-export const formatUnixTimestamp = (milliseconds, formatType) => {
-  const dateObj = new Date(milliseconds);
-  const day = formatType.day(dateObj);
-  const month = formatType.month(dateObj);
-  const year = formatType.year(dateObj);
-  return formatType.format(day, month, year);
-};
+export const formatUnixTimestamp = (timestamp, momentString) =>
+  moment(timestamp)
+    .tz('Europe/London')
+    .format(momentString);
 
 export const isTenHoursAgoOrLess = milliseconds => {
   const now = Date.now();
   return now - milliseconds <= 10 * 60 * 60 * 1000;
 };
+
+// 2019-03-22
+export const longNumeric = 'YYYY-MM-DD';
```

Comment:
It would be good to namespace these values. 
```suggestion
export const formatDateNumeric = 'YYYY-MM-DD';
```